cfme/tests/control/test_bugs.py

```diff
@@ -172,3 +173,25 @@ def _delete_action():
             action.delete()
 
     action.create()
+
+
+@pytest.mark.meta(blockers=[1395965], automates=[1395965])
+def test_delete_all_actions_from_compliance_policy(request):
+    """We should not allow a compliance policy to be saved
+    if there are no actions on the compliance event.
+    Steps:
+        * Create a compliance policy
+        * Remove all actions
+    The policy shouldn't be saved.
+    """
+    policy = VMCompliancePolicy(fauxfactory.gen_alphanumeric())
+
+    @request.addfinalizer
+    def _delete_policy():
+        if policy.exists:
+            policy.delete()
+
+    policy.create()
+    # Until bug is not fixed we will match any character
+    with error.expected(".*"):
+        policy.assign_actions_to_event("VM Compliance Check", [])
```

Comment:
That fails matching a flash message, right?

What about?

```python
from cfme.exceptions import FlashMessageException

with pytest.raises(FlashMessageException):
    policy.assign_actions_to_event("VM Compliance Check", [])
```

@RonnyPfannschmidt @psav ?